pkgs/development/libraries/libvmaf/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, autoconf, automake, intltool, libtool, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  name = "libvmaf";
```

Comment:
should be pname, so the name becomes  `"${pname}-${version}"`
```suggestion
  pname = "libvmaf";
```

Reply:
Changed.